### DISCUSSION TITLE
Window resizing causes a white background flickering when `window-theme` is set to `light`

### DISCUSSION THREAD

**Discussion by eti0** [ghostty-org/ghostty#3840] [Feature Requests, Ideas] [answered]
As you can see on the video below, when `window-theme` is set to `light` in the config file, the resizing behavior shows a white background on the newly resized area before switching to the correct background color.

https://github.com/user-attachments/assets/96aa2a02-6d09-450b-8eab-7506afc4b660

In the video above, I compare iTerm's behavior with Ghostty's. I'd like for the resizing behavior to be similar to iTerm's because I like setting my border color to something a little less light.

**Comment by mitchellh** [comment, marked as answer]
I believe this is a duplicate but will not close until we verify that.
